cypress/e2e/Admin/create/createView.cy.js

```diff
@@ -39,6 +60,25 @@ describe('Create View', () => {
     cy.get(dataCyWrapper(QUESTION_BAR_PREV_CY)).should('be.disabled');
   });
 
+  it('Add questions with empty data', () => {
+    cy.get(dataCyWrapper(ADD_NEW_QUESTION_TITLE_CY)).should('be.visible');
+    fillMultipleChoiceQuestion(newMultipleChoiceData);
+    cy.wait(2000); // Wait for the new question to appear
+    cy.get(`.${QUESTION_BAR_ADD_NEW_BUTTON_CLASSNAME}`).click();
```

Comment:
Add comment like: `add two questions to verify they are added in the order list, cannot check app setting directly`